Src/WitsmlExplorer.Console/ListCommands/ListRisksCommand.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading.Tasks;
+using Spectre.Console;
+using Spectre.Console.Cli;
+using Witsml;
+using Witsml.Data;
+using Witsml.Extensions;
+using Witsml.ServiceReference;
+using WitsmlExplorer.Console.Extensions;
+using WitsmlExplorer.Console.WitsmlClient;
+
+namespace WitsmlExplorer.Console.ListCommands
+{
+    public class ListRisksCommand : AsyncCommand<ListRisksCommand.ListRisksSettings>
+    {
+        public class ListRisksSettings : CommandSettings
+        {
+            [CommandOption("--uidWell")]
+            [Description("Well UID")]
+            [DefaultValue("")]
+            public string WellUid { get; init; }
```

Comment:
Should public classes be nested inside other public classes?

Reply:
Tried (although very briefly) to set the settings classes to a lesser access that the command classes and that didn't work with the generics.

I'm creating a separate PR where I extract the settings classes to individual files instead